erpnext/accounts/report/accounts_receivable/accounts_receivable.py

```diff
@@ -563,7 +565,7 @@ def get_future_payments_from_journal_entry(self):
 				jea.party,
 				jea.party_type,
 				je.posting_date as future_date,
-				sum({0}) as future_amount,
+				sum('{0}') as future_amount,
```

Comment:
Why this change? I think the query is working as it is right now

Reply:
Hi @deepeshgarg007,

Earlier in the account payable report the future payment filter with column was not available.
So, in order to fulfill the change for both account receivable report and account payable report I did this change and its working.

Thanks.

Reply:
@Solufyin I understood the reason for the PR, my concern is regarding the quotes in the query `sum('{0}')`, they were not there before and the report was working fine, why add them now?

Reply:
Hi @deepeshgarg007,

Because of the use of ```sum({0})``` then only the account receivable report worked but the account payable report did not work with the added new feature. So added ```sum('{0}')``` then after checked so both reports are worked properly.

Thanks.

Reply:
Hi @deepeshgarg007 , @nextchamp-saqib,

Is there any update?